class/defaults.yml

```diff
@@ -211,7 +228,9 @@ parameters:
         existingSecret: ${keycloak:database:secretname}
         image:
           registry: quay.io
+        # master renamed to primary later https://github.com/bitnami/charts/commit/7eabc85fd4fae43127228a22829c7ce3fe85c389
         master:
```

Comment:
What is the purpose of this comment? 
Should it be named `master` or `primary`?